api/ocm/tools/transfer/transferhandler/options.go

```diff
@@ -0,0 +1,40 @@
+package transferhandler
+
+import (
+	"github.com/mandelsoft/goutils/errors"
+)
+
+//////////////////////////////////////////////////////////////////////////////
+
+// ConfigOption describes generic non-standard options.
+// Specialized option set implementation map such generic
+//
+//	config to their specialized settings. The format depends
+//
+// on the option target. For example, for spiff it is a spiff
+// script.
```

Comment:
```suggestion
// ConfigOption describes generic non-standard options.
// Specialized option set implementation map such generic
// config to their specialized settings. The format depends
// on the option target. For example, for spiff it is a spiff
// script.
```

the format of this comment seems broken